envfile.sample

```diff
@@ -40,4 +40,5 @@ UVICORN_RELOAD=True
 #DATASET_STORAGE_BASE_URL=s3://jataware-world-modelers-dev/datasets/
 #DOCUMENT_STORAGE_BASE_URL=s3://jataware-world-modelers-dev/documents/
 DOCUMENT_STORAGE_BASE_URL=file:///documents/
-DISABLE_SEMANTIC_HIGHLIGHT=false
\ No newline at end of file
+DISABLE_SEMANTIC_HIGHLIGHT=false
+PLUGINS={"causemos":"src.plugins.causemos.CausemosPlugin","logger":"src.plugins.logging.LoggingPlugin", "sync":"src.plugins.sync.SyncPlugin"}
```

Comment:
These are the values set in the deployed .envfile, but it's never bad to have them as examples here.